src/utils/location.js

```diff
@@ -15,7 +15,7 @@ type IncompleteLocation = {
 
 export function createLocation({
   sourceId,
-  line = 1,
+  line = NaN,
```

Comment:
Instead of setting it to NaN, can we change the type of line to be a number or null? Which wd make more sense. And then we wont pass the line for creating a new location

Reply:
I agree that `NaN` doesn't feel right; null or undefined would be fine with me too.

We should then also see what happens when we click a filename heading in the breakpoints pane.

Ideally we leave scroll position alone unless a specific line has reason for focus, like clicking a breakpoint or pausing.